main.tex

```diff
@@ -809,6 +809,138 @@ \subsection{Inf-sup conditions}
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{Saddle point problems}
+
+A saddle point problem has the form:
+\begin{displaymath}
+    \begin{bmatrix}
+        A & B^{T}\\ 
+        B & 0
+    \end{bmatrix}
+    \begin{bmatrix}
+        u \\ p
+    \end{bmatrix}
+    =
+    \begin{bmatrix}
+        f \\ g
+    \end{bmatrix}.
+\end{displaymath}
+
+To grasp the importance of this formulations, let's first see
+some examples:
+\begin{itemize}
+    \item \textbf{Stokes}: The variational formulation of
+    Stokes reads:
+    \begin{displaymath}
+    \begin{cases}
+        (\nabla u, \nabla v) - (p, \dive v)
+        &=
+        \langle f, v \rangle
+        \quad\forall v
+        \\
+        (\dive u, q) &= 0 \hspace{3em}\forall q
+    \end{cases}
+    \end{displaymath}
+    From where it's clear that \(A=-\Delta\), \(B=\dive\).
+
+    \item \textbf{Darcy (or mixed Poisson)}: We are dealing 
+    with the problem
+    \begin{displaymath}
+        -\Delta u = f
+    \end{displaymath}
+    Introducing the variable \(\sigma = -\nabla u\), the problem
+    now reads:
+    \begin{displaymath}
+    \begin{cases}
+        \dive \sigma &= f \\ 
```

Comment:
dominio en ambas ecuaciones acá también. Para el alineamiento, puede servir \begin{aligned} en vez de cases y usar algo como
    div sigma &= f   && \text{in $\Omega$}
Puedes mantener el lcurly usando  \left\{  antes de 'begin aligned' y \right. al final.